playbook/app/pb_kits/playbook/pb_icon/_icon.tsx

```diff
@@ -152,7 +151,19 @@ const Icon = (props: IconProps) => {
           </span>
         </>
       )
-
+    else if (iconURL)
+      return (
+        <>
+          <span
+              {...dataProps}
+              {...htmlProps}
+              className={classesEmoji}
+              id={id}
+          >
+            <img src={iconURL} />
+          </span>
+        </>
+      )
```

Comment:
When there is a url string for `icon` prop, use that on an image tag to render the SVG.